frontend/webpack.config.js

```diff
@@ -22,10 +21,6 @@ module.exports = {
     ],
     loaders: [
       {
-        test: /\.s?css$/,
-        loader: ExtractTextPlugin.extract('style-loader', `css?sourceMap!sass?sourceMap&${sassPaths}`),
-      },
-      {
```

Comment:
Always good to clean up webpack config as well.

I noticed it still has this plugin in it: `new ExtractTextPlugin('style.css'),`. Can that be deleted too? If so, we could delete the whole dependency on `extract-text-webpack-plugin`.